### DISCUSSION TITLE
Contributing to the wiki?

### DISCUSSION THREAD

**Discussion by jedahan** [kisslinux/repo#324] [Q&A]
Is this discussion forum the best way to start contributing to the wiki?

I'd like to maybe start a `hosting/` or `hardware/` directory to post instructions/logs for installing to different places - think thinkpads, digitalocean droplet, qemu vm. As an example of an install log that could maybe turn into a more complete wiki entry, see https://github.com/kiss-community/kisscommunity.org/wiki/DigitalOcean

**Comment by dylanaraps** [comment]
Pull requests can be opened at https://github.com/kisslinux/website. 

I will be adding Wiki entries about contribution soon (the Wiki's focus, page standards, etc).
